src/rules/no-loud-comments/index.js

```diff
@@ -0,0 +1,37 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("no-loud-comments");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Use // for comments instead of /*"
```

Comment:
As I pointed out in the other rule, the naming convention in stylelint error messages is `"Expected [something] [in some context]"` and `"Unexpected [something] [in some context]"`. We should try to stick to that message format in the future.

https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#determine-violation-messages